tests/lambda_functions/delivery_status_processor_lambda/test_delivery_status_processor_lambda.py

```diff
@@ -18,30 +18,34 @@ def all_path_env_param_set(monkeypatch):
 def event():
     """Generates a sample ALB received Twilio delivery status event object."""
     return {
-        "requestContext": {"elb": {"targetGroupArn": "<DEV TARGET GROUP>"}},
+        "requestContext": {
+            "elb": {
+                "targetGroupArn": ""
+            }
+        },
         "httpMethod": "POST",
-        "path": "/deliverystatus",
+        "path": "/sms/deliverystatus",
         "queryStringParameters": {},
         "headers": {
             "accept": "*/*",
             "connection": "close",
-            "content-length": "227",
+            "content-length": "277",
             "content-type": "application/x-www-form-urlencoded; charset=utf-8",
-            "host": "dev-api.va.gov",
-            "i-twilio-idempotency-token": "50609cf4-07f3-4e3f-ac42-044bd13bbc6c",
+            "host": "api.va.gov",
+            "i-twilio-idempotency-token": "32e709c9-0e95-47c0-8f0d-c6c868cf6241",
             "user-agent": "TwilioProxy/1.1",
-            "x-amzn-trace-id": "Self=<SOME VALUE>",
-            "x-forwarded-for": "<COMMA SEPARTED IPS>",
-            "x-forwarded-host": "dev-api.va.gov:443",
+            "x-amzn-trace-id": "",
+            "x-forwarded-for": "",
+            "x-forwarded-host": "api.va.gov:443",
             "x-forwarded-port": "443",
             "x-forwarded-proto": "https",
             "x-forwarded-scheme": "https",
             "x-home-region": "us1",
-            "x-real-ip": "<SOME IP>",
-            "x-twilio-signature": "<SOME VALUE>",
+            "x-real-ip": "",
+            "x-twilio-signature": "GV6ZwxO2f7qTuUKx94saVKju4XI=",
+            "x-use-static-proxy": "true"
         },
         "body": "U21zU2lkPXRoaXNpc3NvbWVzbXNpZCZTbXNTdGF0dXM9c2VudCZNZXNzYWdlU3RhdHVzPXNlbnQmVG89JTJCMTExMTExMTExMTEmTWVzc2FnZVNpZD1zb21lbWVzc2FnZWlkZW50aWZpZXImQWNjb3VudFNpZD10d2lsaW9hY2NvdW50c2lkJkZyb209JTJCMjIyMjIyMjIyMiZBcGlWZXJzaW9uPTIwMTAtMDQtMDE=",
```

Comment:
@edDocMe360 
Please DM me about this. Not related to your work but almost certainly needs to change.

Reply:
Updated this to the same as is in delivery status. See below.

Reply:
addressed.